### PR TITLE
#171: Fix issue where datepicker field is not user-editable.

### PR DESCRIPTION
**Pull request by JustHereForTheBugReport**

### CODE REVIEW

File: src/datepicker.jsx [hunk 1]
```diff
@@ -157,7 +162,7 @@ var DatePicker = React.createClass( {
           handleClick={this.onInputClick}
           handleEnter={this.hideCalendar}
           setSelected={this.setSelected}
-          clearSelected={this.clearSelected}
```
Comment (martijnrusschen) [inline src/datepicker.jsx]: Should we remove the clearSelected method too?

**Comment by martijnrusschen** [comment]
Could you provide some steps to reproduce the issue you're solving here?

**Comment by martijnrusschen** [comment]
This fixes #171 right?

**Comment by martijnrusschen** [comment]
Ping

**Comment by xrado** [comment]
any plan to merge this in?

**Comment by martijnrusschen** [comment]
@xrado I was hoping on getting a reply on this, no activity from @JustHereForTheBugReport since the initial PR.

**Pull request merged**